.gitignore

```diff
@@ -17,4 +17,4 @@ eclcc.log
 ./helm/examples/azure/sa/env-sa
 ./dockerfiles/platform-build-incremental/hpcc.gitpatch
 .env
-vcpkg.json
+/vcpkg.json
```

Comment:
is this change correct?  Seems and unlikely path.

Reply:
It is specifically for the generated vcpkg.json in the route path - others can exist in the vcpkg_overlays which we would want to keep.